mms/go/cmd/gocqlxapp/main.go

```diff
@@ -0,0 +1,104 @@
+package main
+
+import (
+	"goapp/internal/log"
+	"goapp/internal/scylla"
+
+	"github.com/gocql/gocql"
+	"github.com/scylladb/gocqlx"
+	"github.com/scylladb/gocqlx/qb"
+	"github.com/scylladb/gocqlx/table"
+	"go.uber.org/zap"
+)
+
+var (
+	deleteStmt  string
+	deleteNames []string
+
+	insertStmt  string
+	insertNames []string
+
+	selectStmt  string
+	selectNames []string
+
+	tbl *table.Table
+)
+
+type Record struct {
+	FirstName       string `db:"first_name"`
+	LastName        string `db:"last_name"`
+	Address         string `db:"address"`
+	PictureLocation string `db:"picture_location"`
+}
+
+func init() {
+	m := table.Metadata{
+		Name:    "mutant_data",
+		Columns: []string{"first_name", "last_name", "address", "picture_location"},
+		PartKey: []string{"first_name", "last_name"},
+	}
+	tbl = table.New(m)
```

Comment:
How about `mutantDataTable`? and using it directly - this can be a global var

Reply:
It is global but we can change the name.

Reply:
Removed completely.